### ISSUE TITLE
Positron Web: `POSITRON_MODE` env var showing `desktop` in Terminal

### ISSUE THREAD

**Issue by sharon-wang** [posit-dev/positron#5162]
#### Positron and OS details:

Positron Server Web dev build from `main` branch or any of the latest REH-Web release builds.

#### Interpreter details:

N/A

## Describe the issue:

On Web, `POSITRON_MODE` shows `server` as expected in the R Console and Python Console, but in the Terminal it shows `desktop`.

No issues on Desktop: `POSITRON_MODE` shows `desktop` as expected in the Terminal, R Console and Python Console.

## Steps to reproduce the issue:

1. `echo $POSITRON_MODE` in a Terminal in Positron Web
2. See the output `desktop`, but it should be `server`

## Expected or desired behavior:

`echo $POSITRON_MODE` in a Terminal in Positron Web should return `server`

## Were there any error messages in the UI, Output panel, or Developer Tools console?

No

**Comment by juliasilge** [comment]
Here is where we intend to set that env var:

https://github.com/posit-dev/positron/blob/675b180e2ec25781121c268a6bff162ac9e27857/src/vs/platform/terminal/node/terminalEnvironment.ts#L151

**Comment by sharon-wang** [comment]
We'll need to fix this for the 2024.11.0 release in order for https://github.com/posit-dev/positron/issues/4941 to work.

**Comment by sharon-wang** [comment]
Hmm, I think this was a false alarm. I'm going to play around a bit more, but this might be because the Show Terminal setting doesn't take unless a completely new session in Workbench is created.

**Comment by sharon-wang** [comment]
Okay, so for `kallichoreSupervisor.showTerminal` to take, a user needs to enable it in one Positron session in Workbench and then create a new Positron session. The new Positron session will have the setting enabled and the terminal visible.

False alarm confirmed 👍

**Comment by jmcphers** [comment]
Yes, unfortunately you can't change `hideFromUser` once the terminal is created. :-/
